http_logs/challenges/common/default-schedule.json

```diff
@@ -84,13 +84,13 @@
           "operation": "200s-in-range",
           "warmup-iterations": 500,
           "iterations": 100,
-          "target-throughput": {%- if runtime_script_grok is defined %}5{% else %}33{% endif %}
+          "target-throughput": {%- if runtime_fields is defined %}5{% else %}33{% endif %}
         },
         {
           "operation": "400s-in-range",
           "warmup-iterations": 500,
           "iterations": 100,
-          "target-throughput": {%- if runtime_script_grok is defined %}1{% else %}50{% endif %}
+          "target-throughput": {%- if runtime_fields is defined %}0.3{% else %}50{% endif %}
```

Comment:
Are those changes in `target-throughput` tuned for the performance of the hard used for https://elasticsearch-benchmarks.elastic.co ?

Reply:
We talked about this over video earlier but I'll repeat for the billions of folks not in the chat: these are tuned to night-rally-1 and friends against the slowest combination of runtime fields options. We could totally be more precise here but it'd take up a lot of space and take a while to tune.